checkup.sh

```diff
@@ -0,0 +1,67 @@
+
+#!/bin/bash
+nodev=$(node -v);
+funcv=$(func -v);
+
+echo "🛑 Note: If you've installed node and Azure Functions Core Tools but this script is still saying you haven't, please restart your terminal and run the script again."
+echo "\n"
+echo "--------------------------------";
+echo "Checking if node is installed...";
+echo "--------------------------------";
+if [[ $(node -v) ]]
```

Comment:
I think we can use `${nodev}` and `${funcv}` in the IF-clauses instead of calling `node-v` again.

Same on line 34

Reply:
@emsesc Overall looks really good now - also works in Git Bash as expected